client-java/sql/src/main/java/org/evomaster/client/java/sql/internal/SqlDistanceWithMetrics.java

```diff
@@ -0,0 +1,13 @@
+package org.evomaster.client.java.sql.internal;
+
+public class SqlDistanceWithMetrics {
+
+    public double sqlDistance;
+
+    public int numberOfEvaluatedRows;
+
+    public SqlDistanceWithMetrics(double sqlDistance, int numberOfEvaluatedRows) {
+        this.sqlDistance = sqlDistance;
```

Comment:
should do some checks on input here. eg, throw a `IllegalArgumentException` if any of the inputs is smaller than 0

Reply:
ok. I added these requirements to MongoDistanceWithMetrics too. I also added unit tests for this.